Source/CardStackController.m

```diff
@@ -40,6 +41,7 @@ - (instancetype)init {
     if (!self) return nil;
 
     self.isSeachViewControllerHidden = YES;
```

Comment:
I see that we don't follow this rule :( but we should. 
https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html

> Access Instance Variables Directly from Initializer Methods
Setter methods can have additional side-effects. They may trigger KVC notifications, or perform further tasks if you write your own custom methods.

> You should always access the instance variables directly from within an initialization method because at the time a property is set, the rest of the object may not yet be completely initialized. Even if you don’t provide custom accessor methods or know of any side effects from within your own class, a future subclass may very well override the behavior.

```
- (id)initWithFirstName:(NSString *)aFirstName lastName:(NSString *)aLastName {
    self = [super init];
 
    if (self) {
        _firstName = aFirstName;
        _lastName = aLastName;
    }
 
    return self;
}
```
@hyperoslo/ios